src/Utils.php

```diff
@@ -739,6 +739,11 @@ public static function check_products_for_letterbox( $products ) {
 				// If the product is not found, consider the order not eligible.
 				return false;
 			}
+			
+			$is_digital = $product->is_virtual() || $product->is_downloadable();
+			if( !$is_digital ) {
+				return false;
```

Comment:
This will consider the order not eligible for letterbox, what we should do is ignoring this product 
```suggestion
				continue;
```

you should also keep in your mind that the order could have only one product so if if its virtual this will always consider it as a letterbox because `$total_ratio_letterbox_item` will be 0, so we might need to check if the order have eligible products or not